framework/db/SchemaBuilderTrait.php

```diff
@@ -270,4 +270,24 @@ public function money($precision = null, $scale = null)
         }
         return $this->getDb()->getSchema()->createColumnSchemaBuilder(Schema::TYPE_MONEY, $length);
     }
+
+    /**
+     * @deprecated since version 2.0.11
```

Comment:
Deprecation does not mean that the method description should be dropped. Could you restore it?